src/logger.ts

```diff
@@ -36,6 +39,7 @@ export const minLogLevel = ((): "info" | "warn" | "error" => {
 
 const loggerOptions: LoggerOptions = { name: "command-bot", minLogLevel, logFormat, impl: console }
 
+export type LoggerContext = Pick<Context, "logger">
```

Comment:
yea i see why it could be perceived this way. In some sense the goal it is indeed passing the Logger Context, cuz it will include the parent's scope :)